urbansim/developer/sqftproforma.py

```diff
@@ -581,12 +575,17 @@ def lookup(self, form, df, only_built=True, pass_through=None):
         if only_built:
             df = df.query('min_max_fars > 0 and parcel_size > 0')
 
-        # all possible fars on all parcels
         fars = np.repeat(cost_sqft_index_col, len(df.index), axis=1)
 
         # zero out fars not allowed by zoning
         fars[fars > df.min_max_fars.values + .01] = np.nan
 
+        # same thing for heights
+        heights = np.repeat(heights, len(df.index), axis=1)
+
+        # zero out heights not allowed by zoning
+        fars[heights > df.max_height.values + .01] = np.nan
```

Comment:
I can definitely come up with something more informative ;)  (and will fix the tests that I broke) before merging